Geometry_Engine/Modify/SplitAtPoints.cs

```diff
@@ -77,17 +80,16 @@ public static List<Arc> SplitAtPoints(this Arc arc, List<Point> points, double t
             }
             else
                 result.Add(arc.Clone());
-
             return result;
         }
 
         /***************************************************/
 
 
-        public static List<Arc> SplitAtPoints(this Circle circle, List<Point> points, double tolerance = Tolerance.Distance)
+        public static List<ICurve> SplitAtPoints(this Circle circle, List<Point> points, double tolerance = Tolerance.Distance)
```

Comment:
Can we double check if this needs deprecating rather than just updating? I know `Arc` is an `ICurve` so the scripts might tolerate it, but to be safe can you confirm scripts using the old component (returning a list of `Arc`) will still work with this change (to a list of `ICurve`)? If they don't then the original method will need deprecating.